src/textual/widgets/_header.py

```diff
@@ -112,13 +137,21 @@ def __init__(
         id: str | None = None,
         classes: str | None = None,
     ):
+        """Initialise the header widget.
+
+        Args:
+            show_clock: ``True`` if the clock should be shown on the right of the header.
```

Comment:
Is there a difference between surrounding `True` with a pair of backticks versus two pairs of backticks? I have seen both and haven't understood the difference yet.

Reply:
Good question; I was asking myself the same thing as I was going through the docstrings and seeing a difference. I tested it and from what I can tell there's zero difference; both have the same effect (my uninformed guess here would be it's mkdocstrings or something supporting a couple of other doc systems' styles).